sdk/core/azure-core/src/main/java/com/azure/android/core/util/paging/PagedDataCollection.java

```diff
@@ -51,7 +51,6 @@ public P getFirstPage() {
      * @return a page that matches the given id
      */
     public P getPage(@NonNull String pageId) {
-        Objects.requireNonNull(pageId);
```

Comment:
non-null pageId is for the first page I assume?

Reply:
No. We use pageId as key for caching. Allow null key would be risky. Consider the case when more than one pages return null key, it can get really confusing for the client.